src/metrics.jl

```diff
@@ -660,27 +661,27 @@ function _pairwise!(r::AbstractMatrix, dist::SqEuclidean, a::AbstractMatrix)
         sa2j = sa2[j]
         if threshT <= 0
             @simd for i = (j + 1):n
-                r[i, j] = max(sa2[i] + sa2j - 2 * r[i, j], 0)
+                r[i, j] = eval_end(dist, (max(sa2[i] + sa2j - 2 * r[i, j], 0)))
             end
         else
             for i = (j + 1):n
-                selfterms = sa2[i] + sa2j
-                v = max(selfterms - 2 * r[i, j], 0)
-                if v < threshT * selfterms
-                    v = zero(v)
-                    for k = 1:size(a, 1)
-                        v += (a[k, i] - a[k, j])^2
-                    end
+            selfterms = sa2[i] + sa2j
+            v = max(selfterms - 2 * r[i, j], 0)
+            if v < threshT * selfterms
+                v = zero(v)
+                for k = 1:m
+                    v += (a[k, i] - a[k, j])^2
                 end
-                r[i, j] = v
+            end
+            r[i, j] = eval_end(dist, v)
```

Comment:
Unintended change?

Reply:
No, this is fully intentional. I have merged the two codes for pairwise Euclidean and SqEuclidean, which differed only in the final sqrt (and in a different placement of `@inbounds`. The diff log may look a bit weird though.

Reply:
Sorry, what I mean is that the indentation is wrong.

Reply:
🤦